Initial configuration to build ENIA APP & WEB

FerBueroTrebino opened the pull request:
Initial configuration to build ENIA APP & WEB

FerBueroTrebino committed:
Initial configuration to build ENIA APP & WEB

Initial configuration to build ENIA APP & WEB

FerBueroTrebino committed:
Implement ENIA Splash Screen

Implement ENIA Splash Screen

FerBueroTrebino committed:
Implement ENIA Landing Screen

Implement ENIA Landing Screen

FerBueroTrebino committed:
Implement ENIA Login Screen + Define default server to ENIA

Implement ENIA Login Screen + Define default server to ENIA matrix.codigoi.com.ar

FerBueroTrebino committed:
Implement Menu Drawer ENIA options

Implement Menu Drawer ENIA options

FerBueroTrebino committed:
Implement ENIA settings menu

Implement ENIA settings menu

FerBueroTrebino committed:
Deactivate option to do a videocall on jitsi

Deactivate option to do a videocall on jitsi

FerBueroTrebino committed:
Implement ENIA frequent message dialog

Implement ENIA frequent message dialog

FerBueroTrebino committed:
Implement ENIA Light Theme

Implement ENIA Light Theme

FerBueroTrebino committed:
Change app display name to Asesorias ENIA@Virtual

Change app display name to Asesorias ENIA@Virtual

FerBueroTrebino committed:
Change app Icon Launcher to ENIA

Change app Icon Launcher to ENIA

FerBueroTrebino committed:
Update name and config to web version to ENIA

Update name and config to web version to ENIA

jmhlawatsch commented on lib/components/settings_themes.dart:
```diff
@@ -64,7 +64,8 @@ class ThemesSettingsState extends State<ThemesSettings> {
           },
         ),
-        ListTile(
+        // Not available for ENIA
```
Perfecto el comentario en ingles

FerBueroTrebino committed:
Firebase hosting config

Firebase hosting config

FerBueroTrebino committed:
Update name to ENIA and icons for web

Update name to ENIA and icons for web

FerBueroTrebino committed:
Update factory for best practices

Update factory for best practices

FerBueroTrebino committed:
display frequent message in attach menu for web without function

display frequent message in attach menu for web without function

FerBueroTrebino committed:
Update repo links

Update repo links

FerBueroTrebino committed:
Delete hint text from user text input login

Delete hint text from user text input login

FerBueroTrebino committed:
Replace empty chat for 'Nuevo chat'

Replace empty chat for 'Nuevo chat'

FerBueroTrebino committed:
Fix landing image height so the button is always available on web

Fix landing image height so the button is always available on web

FerBueroTrebino committed:
Fix show frequent messages on web

Fix show frequent messages on web

FerBueroTrebino committed:
Change close session button close text and username input text

Change close session button close text and username input text

FerBueroTrebino committed:
Delete landing screen initial text

Delete landing screen initial text

FerBueroTrebino committed:
Change project name to enia@virtual

Change project name to enia@virtual

FerBueroTrebino committed:
Add option in chat to send videos

Add option in chat to send videos

FerBueroTrebino committed:
Show play button on video messages

Show play button on video messages

FerBueroTrebino committed:
Update description of the web version

Update description of the web version

FerBueroTrebino committed:
Bring frequent message from JSON

Bring frequent message from JSON

FerBueroTrebino committed:
Send video as an attached implementation + Dont allow encryptation on messages

Send video as an attached implementation + Dont allow encryptation on messages

FerBueroTrebino committed:
Implement Fierbase push notifications

Implement Fierbase push notifications

FerBueroTrebino committed:
Update frequent message dialog

Update frequent message dialog

FerBueroTrebino committed:
Improve button separation on landing

Improve button separation on landing

FerBueroTrebino committed:
Change menu chat attach order

Change menu chat attach order

FerBueroTrebino committed:
Implement get frequent messages from plan ENIA API

Implement get frequent messages from plan ENIA API

FerBueroTrebino committed:
Basic implementation of new items on app menu

New item in menu:
* Enia@virtual
* Files
* Formation
* Stars

FerBueroTrebino committed:
Fix menu item enia@virtual

Fix menu item enia@virtual

FerBueroTrebino committed:
Implement get frequent messahes from https

Implement get frequent messahes from https

FerBueroTrebino committed:
Implementation of Change is direct access for Main Group ENia

Implementation of Change is direct access for Main Group ENia

FerBueroTrebino committed:
Improve implementation of get main group list

Improve implementation of get main group list

FerBueroTrebino committed:
Implement Search of main group

Implement Search of main group

FerBueroTrebino committed:
Inlude contact of Group ENIA if the user accept the invitation

Inlude contact of Group ENIA if the user accept the invitation
Put ENIA group always first in the list

FerBueroTrebino committed:
Implement second and third main link + Update menu stats page

Implement second and third main link + Update menu stats page

FerBueroTrebino committed:
Update provisory content for ENIA menu

Update provisory content for ENIA menu

FerBueroTrebino committed:
Fix main group and link fetch

Fix main group and link fetch

FerBueroTrebino committed:
remove imports not used

remove imports not used

FerBueroTrebino committed:
Allow video call

Allow video call

FerBueroTrebino committed:
Add Third link list comparision (groups provincias ENIA)

Add Third link list comparision (groups provincias ENIA)

FerBueroTrebino committed:
Update deprecated package from fluffy

Update deprecated package from fluffy

FerBueroTrebino committed:
Include Version in splash and settings

Include Version in splash and settings

FerBueroTrebino committed:
Update stats content in menu

Update stats content in menu

FerBueroTrebino committed:
Fix open one room on multiple taps over presence item

Fix open one room on multiple taps over presence item

FerBueroTrebino committed:
Upgrade version

Upgrade version

FerBueroTrebino committed:
Update app and SDK version

Update app and SDK version

FerBueroTrebino committed:
cleaning the code

cleaning the code

FerBueroTrebino committed:
Add MinSalud and Unicef Logos to credits and login + fix search enia contacts

Add MinSalud and Unicef Logos to credits and login + fix search enia contacts

FerBueroTrebino committed:
Update Version

Update Version

FerBueroTrebino committed:
Update Stats Menu Pages

Update Stats Menu Pages

FerBueroTrebino committed:
Merge branch 'master' of https://github.com/cottic/enia-virtual

FerBueroTrebino committed:
Merge branch 'master' into dev

FerBueroTrebino committed:
Fixes post merge

Fixes post merge

FerBueroTrebino committed:
Fix custom links and main group contacts

Fix custom links and main group contacts

FerBueroTrebino committed:
Fix Send Video in Chat after merge and package changed

Fix Send Video in Chat after merge and package changed

FerBueroTrebino committed:
Finish config/fixes after merge

Finish config/fixes after merge

FerBueroTrebino committed:
Comment set status button for Enia version

Comment set status button for Enia version

FerBueroTrebino committed:
Implement translations to new enia text

Implement translations to new enia text

FerBueroTrebino committed:
Implement EniaClient class and get frequent asqued

Implement EniaClient class and get frequent asqued

FerBueroTrebino committed:
basic implementation of stats filters

basic implementation of stats filters

FerBueroTrebino committed:
Improve stats implementation

Improve stats implementation

FerBueroTrebino committed:
Improve implementation stats dashboard + create neccesary widgets + charts

Improve implementation stats dashboard + create neccesary widgets + charts

FerBueroTrebino committed:
Add indicators to line and bar charts

Add indicators to line and bar charts

FerBueroTrebino committed:
Implement app settings from json + Pie chart from json

Implement app settings from json + Pie chart from json

FerBueroTrebino committed:
Implement Line Chart from Json

Implement Line Chart from Json

FerBueroTrebino committed:
Implementation Bar Chart  + Define folders structure for stats

Implementation Bar Chart  + Define folders structure for stats

FerBueroTrebino committed:
Implement Card Chart

Implement Card Chart

FerBueroTrebino committed:
update chat list and hosting file

update chat list and hosting file

FerBueroTrebino committed:
Fix routing json + name files

Fix routing json + name files

FerBueroTrebino committed:
Fix to show dashboard on mobile

Fix to show dashboard on mobile

FerBueroTrebino committed:
Make available to change password

Make available to change password

FerBueroTrebino committed:
Internazionalize text + format double quotes + remove unused imports

Internazionalize text + format double quotes + remove unused imports

FerBueroTrebino committed:
Improving UX of dashboard

Improving UX of dashboard

FerBueroTrebino committed:
Get Card Chart data From ENIA API

Get Card Chart data From ENIA API

FerBueroTrebino committed:
Implement Pie Single Chart

Implement Pie Single Chart

FerBueroTrebino committed:
Implement Dashboard Constants Styles

Implement Dashboard Constants Styles

FerBueroTrebino committed:
Fix calculate remaining porcentage in pie single chart

Fix calculate remaining porcentage in pie single chart

FerBueroTrebino committed:
Change dashboard font to Encode Sans

Change dashboard font to Encode Sans

FerBueroTrebino committed:
Fix dashboard header

Fix dashboard header

FerBueroTrebino committed:
Implement Pie Chart with boubles

Implement Pie Chart with boubles

FerBueroTrebino committed:
Set version to 1.7.0 and update package fl chart

Set version to 1.7.0 and update package fl chart

FerBueroTrebino committed:
Fix display Pie Chart Info

Fix display Pie Chart Info

FerBueroTrebino committed:
Fix show links at top of list

Delete displayShareStatus, not a functionlity in ENIA.
Delete Display presences

FerBueroTrebino committed:
Bar Chart adjustments

Bar Chart adjustments

FerBueroTrebino committed:
delete unused json files + update charts url

delete unused json files + update charts url

FerBueroTrebino committed:
finish implementation line chart

finish implementation line chart

FerBueroTrebino committed:
Finish implementation Pie Chart

Finish implementation Pie Chart

FerBueroTrebino committed:
Finish implementation pie singe chart

Finish implementation pie singe chart

FerBueroTrebino committed:
Finish implementation dashboard 01

Finish implementation dashboard 01

FerBueroTrebino committed:
Update version to 1.7.2

Update version to 1.7.2

jmhlawatsch committed:
Finals setting to Board0

FerBueroTrebino committed:
Revert "Finals setting to Board0"

This reverts commit f905321a1f5a4d509a9471965c5e74f6e4ab21d9.

FerBueroTrebino committed:
Revert "Revert "Finals setting to Board0""

This reverts commit 89f24624ca3de5b3a7d50716d3b6f1d570c5edf7.

FerBueroTrebino committed:
Finish implementation Provincia Filter

Finish implementation Provincia Filter

FerBueroTrebino committed:
Adjust styles pie chart and pie single chart

Adjust styles pie chart and pie single chart

FerBueroTrebino committed:
Final adjust in charts UI

Final adjust in charts UI

FerBueroTrebino committed:
Add no info messages to line and bar chart

Add no info messages to line and bar chart

jmhlawatsch committed:
New menu: Maps and 1 link to map in page

jmhlawatsch committed:
Se incluyen el resto de los 8 mapas

jmhlawatsch committed:
Include 10 dashboards and pages

jmhlawatsch committed:
Update setting dashboard definitions

jmhlawatsch committed:
Dashboard 1 2 3 4 5

FerBueroTrebino committed:
Implement message when is 0 in Pie Single Chart

Implement message when is 0 in Pie Single Chart

FerBueroTrebino committed:
Implement error message when data is 0 in Pie Chart

Implement error message when data is 0 in Pie Chart

FerBueroTrebino committed:
Implement Enia Custom icons

Implement Enia Custom icons

FerBueroTrebino committed:
Final implementation Enia Icons in Card Chart

Final implementation Enia Icons in Card Chart

FerBueroTrebino committed:
Fix card display

Fix card display

FerBueroTrebino committed:
Modify how to get frequent messages, no more from api, now from assets

Modify how to get frequent messages, no more from api, now from assets

FerBueroTrebino committed:
implement message for screen resolution dashboard

If the width of the screen is less than 1000 show a message indicating that you should enter in desktop

jmhlawatsch committed:
ESI amd DIAJU dashboard 1st version

FerBueroTrebino committed:
Fix enia icons adolescente y ninios head was missing

Fix enia icons adolescente y ninios head was missing

FerBueroTrebino committed:
Fix remain translations and unused imports

Fix remain translations and unused imports

FerBueroTrebino committed:
Fix bottom titles display

Fix bottom titles display

jmhlawatsch committed:
Update frequentMessagesInfo

FerBueroTrebino committed:
Modify how the bar chart get the height

Modify how the bar chart get the height

FerBueroTrebino committed:
Fix position circular progress indicator in frequent messages

Fix position circular progress indicator in frequent messages

FerBueroTrebino committed:
Update version to 1.8.0

Update version to 1.8.0

FerBueroTrebino committed:
Update version to match Google Play

Update version to match Google Play

FerBueroTrebino committed:
Fix bar chart duplicate bars

Fix bar chart duplicate bars

jmhlawatsch committed:
Merge pull request #2 from cottic/update-extend-menu-options

Update extend menu options
